Vcom/Functions/VCM_Functions/fn_FrmChnge.sqf

```diff
@@ -20,7 +20,7 @@ _unit = _this;
 _group = group _unit;
 
 //Vehicular groups in "SAFE" behaviour will move in convoys
-if (!isNull objectParent _unit && {behaviour _unit isEqualTo "SAFE"}) exitWith 
+if (!isNull objectParent _unit && {behaviour _unit == "SAFE"}) exitWith 
```

Comment:
isEqualTo is faster because it is case sensitive... and behaviour only outputs "SAFE"